src/app/components/feed/SelectList.module.css

```diff
@@ -0,0 +1,5 @@
+.select-list-container {
```

Comment:
I added this and got rid of a `style` prop that was styling the `SelectList`. Should be more consistent with other components now.